docs/developing-flows/flow-structure.md

```diff
@@ -0,0 +1,115 @@
+
+Over time, that can lead to a mess of nodes and wires that make it hard to find particular parts of the flow.
+
+Putting some thought into how to structure your flows at the start of any development project can help keep them organised and make them easier to maintain.
+
+The main method of organising flows in Node-RED is by separating them across multiple tabs within the editor. There are a few different strategies that can be used to do that.
+
+If you can identify separate logical components of your application, consider putting them on separate tabs.
+
+For a home-automation application, you could put the flow logic for each room on a separate tab to reflect the physical space. Or you may want to separate the flows based on function - so all lighting-related flows go on one tab and heating on another.
+
+If you are building an HTTP API backend, each tab could represent a separate type of resource the API accesses.
+
+The goal should be to make it easy to “read” an individual flow from start to finish. Keeping it on a single tab can help do that.
+
+Another consideration is whether you are working alongside other developers on the same Node-RED application. It is much easier to manage the merging of changes if they are on separate tabs. If you have developers with different roles or specialisations, consider how that may affect how your flows are organised.
+
+
+### Making reusable flows
+
+As you build your flows, you may find some common parts that you want to reuse in multiple places. You should avoid having multiple copies of those common parts spread across your flows as they become harder to maintain - you end up with multiple places to apply fixes and could easily overlook one.
+
+Node-RED provides two different ways of creating reusable flows - Links nodes and Subflows.
+
+Link nodes let you create a flow that can jump between tabs in the editor - they add a virtual wire from the end of one flow to the start of another.
+
+
+<div style="width: 300px" class="figure">
+  <img src="images/placeholder.png" alt="Link nodes">
+  <p class="caption">Placeholder for image: Link nodes</p>
+</div>
+
+Subflows let you create a new node in the palette whose internal implementation is described as a flow. You can then add new instances of the subflow wherever you would a normal node.
+
+<div style="width: 300px" class="figure">
+  <img src="images/placeholder.png" alt="Link nodes">
+  <p class="caption">Placeholder for image: Link nodes</p>
+</div>
+
+
+There are some important differences between the two approaches. Link nodes cannot be used in the middle of a flow, where messages are passed over the link and then return when the other flow completes. They can only be used to start or end a flow. They can also be connect to more than one other link node. This lets you pass messages out to multiple other flows, or have multiple flows pass messages into a single flow.
+
```

Comment:
Links can also be used within a page to declutter if you have lots of wires feeding to or from a single point, or flying back from right to left.